products-and-docs/apis/commerce/index.md

```diff
@@ -2922,3 +2922,205 @@ commerce/{version}/shopping/carts/{cartId}/products.{format}
 Same as the [Get Cart API](#get-cart).
 
 
+
+## Select Payments
+{: .article .console-link #select-payments}
+
+**Method:** PATCH
```

Comment:
If the apiDescription listed these methods as POST, then change this as POST, otherwise, change the above.